### PR TITLE
Feat add substation properties fix3

### PR DESCRIPTION
**Pull request by laurgarn**

### CODE REVIEW

File: src/utils/rest-api.js [hunk 1]
```diff
@@ -1887,7 +1887,7 @@ export function createSubstation(
         substationCountry: substationCountry === '' ? null : substationCountry,
         properties: asObj,
     });
-    console.debug('createSubstation body', properties, body);
+    console.debug({ body });
```
Comment (AbdelHedhili) [inline src/utils/rest-api.js:1890]: a little message can be nice to understand better what we see in the console

**Review by AbdelHedhili — APPROVED** [review]
code ; ok 
not tested

File: src/utils/rest-api.js (hunk 1 above)
Comment (laurgarn) [reply src/utils/rest-api.js:1890]: There is `console.info('Creating substation creation');` some lines above, so I thought it could suffice.

Comment (AbdelHedhili) [reply src/utils/rest-api.js:1890]: I was talking about the fact that we don't know that it's the body, see the previous message that you changed it was better

Comment (AbdelHedhili) [reply src/utils/rest-api.js:1890]: I was talking about the fact that we don't know that it's the body, see the previous message that you changed it was better

**Review by dbraquart — APPROVED** [review]

**Pull request merged**